app/routines/update_user_contact_info.rb

```diff
@@ -46,11 +46,14 @@ def call
 
       user.salesforce_contact_id = sf_contact.id
 
-      SecurityLog.create!(
-        user: user,
-        event_type: :user_contact_id_updated_from_salesforce,
-        event_data: { contact_id: sf_contact.id }
-      )
```

Comment:
It's probably fine to save this when the contact is actually updated, just don't save it when there are no changes.